### ISSUE TITLE
[Firefox] Present the graph of visual metrics in profiler without the need of profiler screenshots. 

### ISSUE THREAD

**Issue by sefeng211** [sitespeedio/browsertime#1214]
Using browsertime to collect firefox profiles allow the profiler to present a visual metrics graph, [here's an example](https://profiler.firefox.com/public/7788671ed67417b9acd056310b99715892ffefe1/calltree/?globalTrackOrder=5-6-7-4-0-1-2-3&hiddenGlobalTracks=1-2&localTrackOrderByPid=21091-1-2-0~21094-0~21092-0~21093-0-1~&thread=4&v=4), but this graph would only be presented if you have `screenshots` as one of the enabled profiler feature `(--firefox.geckoProfilerParams.features "screenshots")`. 

Maybe we should allow showing this graph without the need of this feature?

**Comment by soulgalore** [comment]
Hi @sefeng211 very cool, I did't know it was included in the profiler. So what need to be done in Browsertime, I didn't fully get that?

**Comment by brennie** [comment]
@soulgalore I believe what @sefeng211 is suggesting is that instead of doing 

```
browsertime.js --video --firefox.geckoProfiler --firefox.geckoProfilerParams.features "screenshots"
```

to generate a profile with screenshots AND use visual metrics, it should be possible to do:

```
browsertime.js --video --firefox.geckoProfiler
```
and use the frames from the video as the screenshots in the profile.

**Comment by sefeng211** [comment]
Yup exactly ^! Basically just remove the requirement of `--firefox.geckoProfilerParams.features "screenshots"`

**Comment by soulgalore** [comment]
Yes, feel free to change (and merge), you guys are the experts on Firefox stuff so please go ahead :)

**Comment by soulgalore** [comment]
Nothing happened here from the Firefox side for 6 years so closing.
